optional_plugins/html/avocado_result_html/__init__.py

```diff
@@ -248,8 +248,7 @@ def _render(self, result, output_path):
             else:
                 from pystache import view
                 v = view.View(template, context)
-                report_contents = v.render('utf8')  # encodes into ascii
-                report_contents = codecs.decode("utf8")  # decode to unicode
+                report_contents = v.render('utf8')
```

Comment:
Well I'd prefer try/catch to convert it to utf-8 and when that fails I'd go with normal version. If everything fails (as we expect utf-8 chars to be present) I'd only log warning informing about html report failure, rather than failing the whole avocado job.